packages/domainobjs/package.json

```diff
@@ -47,5 +50,6 @@
     "lines": ">50%",
     "functions": ">50%",
     "statements": ">50%"
-  }
+  },
+  "gitHead": "161d332ca43646e80d7ef9abc739b20ceb5073b6"
```

Comment:
Why do we need this?

Reply:
oh that was added automatically when testing publish, will remove